rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -1244,7 +1244,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|!REQUEST_COOKIES:/_pk_ref/|REQU
 # SecRuleUpdateTargetById 942430 "!ARGS:foo"
 #
 
-SecRule ARGS_NAMES|ARGS|XML:/* "@rx ((?:[\~\!\@\#\$\%\^\&\*\(\)\-\+\=\{\}\[\]\|\:\;\"\'\´\’\‘\`\<\>][^\~\!\@\#\$\%\^\&\*\(\)\-\+\=\{\}\[\]\|\:\;\"\'\´\’\‘\`\<\>]*?){12})" \
+SecRule ARGS_NAMES|ARGS|XML:/* "@rx ((?:[~!@#\$%\^&\*\(\)\-\+=\{\}\[\]\|:;\"'´’‘`<>][^~!@#\$%\^&\*\(\)\-\+=\{\}\[\]\|:;\"'´’‘`<>]*?){12})" \
```

Comment:
Thanks for fixing this!

We can actually go further here and remove all backslashes except the ones before
"
^
]
-

(this applies to the regexes below, too)

Reply:
`"` needs to be escaped, and for consistency I'd like to escape `]` too.
`-` should be escaped as well, otherwise `-` will not match and instead `*` will be included.
Since distinct engines might have subtle differences I think for the time being this is a good compromise.